docs/go/how-to-set-activity-timeouts-in-go.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
This file is meant to be for Activity timeouts - but it looks like Workflow timeout info was copied in. Appears to be a duplicate of how-to-set-workflow-timeouts-in-go